kornia/enhance/jpeg.py

```diff
@@ -499,8 +522,9 @@ def jpeg_codec_differentiable(
     )
     # Clip coded image
     image_rgb_jpeg = differentiable_clipping(input=image_rgb_jpeg, min_val=0.0, max_val=255.0)
-    # Back to original shape
-    # image_rgb_jpeg = image_rgb_jpeg.view(original_shape)
+    # Crop the image again to the original shape if needed
+    if (h_pad != 0) or (w_pad != 0):
+        image_rgb_jpeg = image_rgb_jpeg[..., :-h_pad, :-w_pad]
```

Comment:
Can we remove the if-else block here since we are trying to support torch.compile and ONNX-ish stuff.

Reply:
Done in [35e8d50](https://github.com/kornia/kornia/pull/2865/commits/35e8d507796a8b4f1baa36a5b170c6adb8a4931b)